backend/user-service/app/routes/user.router.ts

```diff
@@ -0,0 +1,19 @@
+import { Router } from "express";
+import { userController } from "../controllers/user.controller";
+
+export const userRouter = Router();
+
+/* Get user by ID */
+userRouter.get('/id/:userId', userController.getById)
+
+/* Get user by email */
+userRouter.get('/email/:email', userController.getByEmail)
+
+/* Create user */
+userRouter.post('/', userController.post)
+
+/* Update user */
+userRouter.put('/', userController.put)
```

Comment:
```suggestion
userRouter.put('/id/:userId', userController.put)
```
ps i wrote wrongly in defining the spec

Reply:
Resolved fe8821837b44cb190648ef17af038ec332c3a2b8